control/table.go

```diff
@@ -71,20 +62,41 @@ func (ct *Table) controlName() (name string, err error) {
 
 // Fill inserts the OpenControl justifications into the table. Note this modifies the `table`.
 func (ct *Table) Fill(openControlData opencontrols.Data) (err error) {
-	roleCell, err := ct.responsibleRoleCell()
+	roleCell, err := findResponsibleRole(ct)
 	if err != nil {
 		return
 	}
 
-	existingContent := roleCell.Content()
 	control, err := ct.controlName()
 	if err != nil {
 		return
 	}
 
 	roles := openControlData.GetResponsibleRoles(control)
-	content := fmt.Sprintf("%s %s", existingContent, roles)
-	roleCell.SetContent(content)
+	roleCell.setValue(roles)
```

Comment:
Not blocker, but we may want to consider if we should check to see if there's existing content that differs, since this could wipe away someone's work if people have been working in both the YAML _and_ the Word doc (hence the need for the diff in the first place). Not sure what the behavior should be.

/cc @brittag (let me know if you need more context here)